delta/imagery/sources/tiff.py

```diff
@@ -318,8 +318,8 @@ def __init__(self, path, width, height, num_bands=1, data_type=gdal.GDT_Byte, #p
 
         # Constants
         options = ['COMPRESS=LZW', 'BigTIFF=IF_SAFER', 'INTERLEAVE=BAND']
-        options += ['BLOCKXSIZE='+str(self._tile_height),
-                    'BLOCKYSIZE='+str(self._tile_width)]
+        options += ['BLOCKXSIZE='+str(self._tile_width),
```

Comment:
@ScottMcMichael It is supposed to set the same block size here in this line you changed. It passed the unit test previously. You don't think this broke the test?

Reply:
I guess that functionality was added at some point.  This particular bug I fixed was being compensated for by other bugs which hopefully I caught all of.  I re-added the test and with all the fixes it passes.